pkg/kp/rcstore/rc.go

```diff
@@ -0,0 +1,95 @@
+package rcstore
+
+import (
+	"github.com/square/p2/pkg/labels"
+	"github.com/square/p2/pkg/pods"
+)
+
+type ReplicationControllerDesires struct {
+	replicasDesired int
+	disabled        bool
+}
+
+type ReplicationControllerData struct {
+	// It is expected that these fields will never be changed after an RC is created.
+	id           string
+	manifest     *pods.Manifest
+	nodeSelector labels.Selector
+	podLabels    labels.Set
+
+	watcher desireWatcher
+
+	// It is expected that these fields can be changed.
+	ReplicationControllerDesires
```

Comment:
I would argue that mutability vs immutability isn't a great reason to separate them.